torch/_C/_distributed_c10d.pyi

```diff
@@ -27,18 +28,48 @@ class GradBucket:
     def set_buffer(self, tensor: Tensor) -> None: ...
     def parameters(self) -> List[Tensor]: ...
 
+kDefaultBucketBytesCap = (
+    25 * 1024 * 1024
+)  # kDefaultBucketBytesCap from reducer.hpp
+kDefaultFirstBucketBytes = (
+    1024 * 1024
+)  # kDefaultFirstBucketBytes from reducer.hpp
+
 class Reducer:
     def __init__(
         self,
         params: List[Tensor],
         bucket_indices: List[List[int]],
+        per_bucket_size_limits: List[int],
         process_group: ProcessGroup,
-        expect_sparse_gradients: List[bool],
-        bucket_bytes_cap: int,
-        find_unused_parameters: bool,
-        gradient_as_bucket_view: bool,
+        expect_sparse_gradients: List[bool] = [],
+        bucket_bytes_cap: int = kDefaultBucketBytesCap,
```

Comment:
instead of having to hardcode the constant here (where it can diverge), you can use an ellipsis instead